content/doc/book/pipeline/running-pipelines.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: section
+title: Running Pipelines
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+endif::[]
+
+== Running a Pipeline
+
+=== Multibranch
+
+See <<multibranch, the Multibranch documentation>> for more information.
```

Comment:
This looks like reasonable minimal content.  

I don't want to block a declarative release to fill these in.  But I don't want them to get completely lost either. Could you file a WEBSITE ticket pointing to this page as needing work and then add comment in here to point to that issue?

Reply:
Will do.